requirements.txt

```diff
@@ -1,5 +1,3 @@
-docker-compose==1.7.0
-docker-py==1.8.0
-Jinja2==2.8
-PyYAML==3.11
-requests==2.7.0
+docker-compose>=1.7.0
+Jinja2>=2.8
+PyYAML>=3.11
```

Comment:
Talked with @gregdek. Right now the best argument against having an RPM is that Ansible-Container is not stable enough to warrant having an RPM. We'll post up the work we have done to create an Ansible Container RPM so that work can be picked up when the time is right.